aptos-move/framework/move-stdlib/sources/configs/config_for_next_epoch.move

```diff
@@ -0,0 +1,78 @@
+/// This wrapper helps store an on-chain config for the next epoch.
+///
+/// Once reconfigure with DKG is introduced, every on-chain config `C` should do the following.
+/// - Support async update when DKG is enabled. This is typically done by 3 steps below.
+///   - Implement `C::set_for_next_epoch()` using `upsert()` function in this module.
+///   - Implement `C::on_new_epoch()` using `extract()` function in this module.
+///   - Update `0x1::reconfiguration_with_dkg::finish()` to call `C::on_new_epoch()`.
+/// - Support sychronous update when DKG is disabled.
+///   This is typically done by implementing `C::set()` to update the config resource directly.
+///
+/// NOTE: on-chain config `0x1::state::ValidatorSet` implemented its own buffer.
+module std::config_for_next_epoch {
+
+    use std::signer::address_of;
+
+    const ESTD_SIGNER_NEEDED: u64 = 1;
+    const ERESOURCE_BUSY: u64 = 2;
+
+    /// `ForNextEpoch<T>` under account 0x1 holds the config payload for the next epoch, where `T` can be `ConsnsusConfig`, `Features`, etc.
+    struct ForNextEpoch<T> has drop, key {
```

Comment:
Why not use a SimpleMap here with the key being the type of configs and the value being vector<u8>? Then you wouldn't need to add every single config as a new resource + need the 0x1's signer

Reply:
why vector? did you mean `SimpleMap<String, Any>`

Reply:
oh, Any is not possible since here is `move-stdlib` and `Any` is in its dependent: `aptos-framework`.